admin_ui/backend/tests/test_export_sanitization.py

```
@@ -0,0 +1,91 @@
+    monkeypatch.setattr(config_mod.settings, "ENV_PATH", str(tmp_path / ".env"))
+    app = FastAPI()
+    app.include_router(config_mod.router, prefix="/api/config")
+    no_env_client = TestClient(app)
+
+    resp = no_env_client.get("/api/config/export?include_secrets=true")
+    assert resp.status_code == 200
+    zf = zipfile.ZipFile(io.BytesIO(resp.content))
+    names = set(zf.namelist())
+    assert ".env" not in names, ".env must not appear when it does not exist on disk"
+    readme = zf.read("EXPORT_README.txt").decode()
+    included_line = next((l for l in readme.splitlines() if l.startswith("Included:")), "")
```

Ruff (0.15.15)

[error] Ambiguous variable name: `l`

(E741)